packages/ui-kit/src/components/TimeSeries/utils.ts

```diff
@@ -255,12 +255,18 @@ export function getScales(options: GetScalesOptions) {
       ...scalesBase.x,
       type: 'timeseries',
       time: {
+        isoWeekday: true,
         unit: getGranularityBasedUnit(granularity)
+      },
+      adapters: {
+        date: {
+          zone: options.zone
+        }
       }
     }
   }
 
-  const currentFormatScales = isFormatted || isStatic ? customFormatScales : autoFormatScales
+  const currentFormatScales = isFormatted ? customFormatScales : autoFormatScales
```

Comment:
IMO, we should not care whether the data is static or not when deciding to display the automatically labeled time series.